src/test/java/com/qaprosoft/carina/demo/mfp/MobileTest.java

```diff
@@ -307,4 +304,34 @@ public void twoPlansSelectTest() {
                 String.format("New plan \"%s\" isn't present", secondPlan.getPlanName()));
     }
 
+    @Test
+    @MethodOwner(owner = "Hostiuk")
+    @TestRailCases(testCasesId = "12")
+    @TestLabel(name = "feature", value = {"mobile", "regression"})
+    public void amountOfCreatedFoodChangeTest() {
+        LoginPageBase loginPage = initPage(getDriver(), LoginPageBase.class);
+        loginPage.login(R.TESTDATA.get("email"), R.TESTDATA.get("password"));
+
+        CommonPageBase commonPage = initPage(getDriver(), CommonPageBase.class);
+        DashboardPageBase dashboardPage = (DashboardPageBase) commonPage.clickBottomNavigatorButton(
+                BottomNavigatorButtons.DASHBOARD);
+        Assert.assertTrue(dashboardPage.isPageOpened(), "Dashboard page isn't opened");
+        UserProfilePageBase userProfilePage = dashboardPage.openUserProfile();
+        Assert.assertTrue(userProfilePage.isPageOpened(), "User Profile page isn't opened");
+        MyItemsPageBase myItemsPage = (MyItemsPageBase) userProfilePage.openTabPage(
+                UserProfilePageBase.TabButtons.MY_ITEMS);
+        Assert.assertTrue(myItemsPage.isPageOpened(), "My Items page isn't opened");
+        String initialFoodCount = myItemsPage.getItemTitle(MyItemsCreateButtons.FOODS);
+        int expectedFoodCount = (Integer.parseInt(initialFoodCount.replaceAll("[^0-9]", "")) + 1);
+        CreateFoodPageBase createFoodPage = (CreateFoodPageBase) myItemsPage.clickCreateButton(MyItemsCreateButtons.FOODS);
+        Assert.assertTrue(createFoodPage.isPageOpened(), "Create Food page isn't opened");
+        createFoodPage.fillAllFields("test", "test", 1, "g", 1);
+        createFoodPage.clickNextButton();
+        createFoodPage.typeNutritionFact(CreateFoodNutritionFacts.CALORIES, 1);
+        createFoodPage.clickSaveButton();
+        createFoodPage.clickSaveButton();
+        createFoodPage.clickNoThanksButton();
```

Comment:
for this purposes usually we create one method - eg createSimpleFood or createDefaultFood and the other method - createFoodWithParameters. in your case there should be createFoodWithParameters("test", "test", 1, "g", 1) - and all the action has to be inside the method. when you code you do not have to think what button should be tapped, for example, after clickSaveButton, you should take care of it inside createFoodWithParameters method